File: src/main/java/eu/happybe/openapi/mysql/DatabaseData.java
package eu.happybe.openapi.mysql;

import lombok.Getter;

public class DatabaseData {

    public static final String DATABASE = "HappyBE_Test";

    @Getter
    private static String host;
    @Getter
    private static String user;
    @Getter
    private static String password;

    public static void update(String host, String user, String password) {
        DatabaseData.host = host;
        DatabaseData.user = user;
        DatabaseData.password = password;
    }
}

File: src/main/java/eu/happybe/openapi/mysql/DatabaseData.kt
package eu.happybe.openapi.mysql

import lombok.Getter

object DatabaseData {
    const val DATABASE = "HappyBE"

    @Getter
    private var host: String? = null

    @Getter
    private var user: String? = null

    @Getter
    private var password: String? = null
    fun update(host: String?, user: String?, password: String?) {
        DatabaseData.host = host
        DatabaseData.user = user
        DatabaseData.password = password
    }
}